gslib/tests/testcase/integration_testcase.py

```diff
@@ -511,6 +513,8 @@ def CreateBucket(self,
           True.
       bucket_policy_only: If True, set the bucket's iamConfiguration's
           bucketPolicyOnly attribute to True.
+      name_prefix: Unicode string to be prepended to bucket_name
```

Comment:
bucket_name_prefix, bucket_name_suffix

Also, it's worth noting whether these do or don't get added into the name if `bucket_name` is empty/not passed in.

Reply:
Good catch! Will fix names and also make sure that it's prepended & appended when bucket names aren't specified.

Reply:
It looks like if a name isn't supplied, the else block on (this revision's) line 542 should still give it the base name 'bucket' like before with prefix and suffix (once I've fixed the method semantics) :)